apps/src/storage/dataBrowser/dataVisualizer/Snapshot.jsx

```diff
@@ -136,7 +137,13 @@ class Snapshot extends React.Component {
         >
           <div ref="snapshot">
             <h1>{this.props.chartTitle}</h1>
-            <img style={{maxHeight: '50vh'}} src={this.state.imageSrc} />
+            <img
+              style={{maxHeight: '50vh'}}
+              src={this.state.imageSrc}
+              alt={`The ${this.props.chartTypeName} of ${
```

Comment:
this (and the other alt text) should be translatable

Reply:
gahhh yes thank you!